content/goku-kun/introduction-to-adts-in-javascript.md

```diff
@@ -67,56 +67,56 @@ Linked-lists are made up of a sequence of elements (called nodes, refer the left
+search(element); // Searches given element and returns true if the element is found in linked-list
+sort(order); // Sorts the linked-list in the given order (ascending/descending)
+retrieve(index); // Returns the element stored at the given index location
 ```
 
 ## Stack ADTs
 
-**Stacks** are linear data structures, in which data is entered and removed from _only a single point_. This point is called the **top** of the stack. It follows the last-in, first-out (abbreviated as LIFO) format for storing and discarding data. This means that the last element added to the top of the stack is the first element that will be removed from the stack. There is no other way to access other elements in the stack but the element that is at the top of the stack.
+**Stacks** are linear data structures in which data is entered and removed from _only a single point_. This point is called the **top** of the stack. It follows the last-in, first-out (L I F O) format for storing and discarding data. This means that the last element added to the top of the stack is the first element that will be removed from the stack. There is no other way to access other elements in the stack but the element that is at the top of the stack.
 
 ![Stack ADT](https://raw.githubusercontent.com/Codecademy/ugc/main/content/goku-kun/stack-adt.png)
 
 A stack ADT supports the following operations:
 
 ```js
-push(element); // inserts a new element at the top of the stack
-pop(); // removes the element stored at the top of the stack and returns it
-peek(); // returns the top element without removing it from the stack
+push(element); // Inserts a new element at the top of the stack
+pop(); // Removes the element stored at the top of the stack and returns it
+peek(); // Returns the top element without removing it from the stack
 ```
 
 ## Queue ADTs
 
-**Queues** are linear data structures, in which data is inserted from one end and removed from the other end. The place where the data is inserted in the queue is called the **rear** end of the queue and this insertion operation is called **enqueue**. Data can be removed from the **front** of the queue and this deletion operation is called **dequeue**. It follows the first-in, first-out (abbreviated as FIFO) configuration for storing and removing data. This means that the data that was first to enter the queue is also first to leave the queue.
+**Queues** are linear data structures, in which data is inserted from one end and removed from the other end. The place where the data is inserted in the queue is called the **rear** end of the queue and this insertion operation is called **enqueue**. Data can be removed from the **front** of the queue and this deletion operation is called **dequeue**. It follows the first-in, first-out (FIFO) configuration for storing and removing data. This means that the data that was first to enter the queue is also first to leave the queue.
```

Comment:
I originally suggested this idea because I'd recently come across [this source](https://accessiblewebsiteservices.com/accessible-images-how-to-handle-alt-text-acronyms/) that describes delimiting acronyms with spaces in _alt text_ in images.

This isn't a huge requirement by any means, but something a11y-related to consider with regards to screen-readers that may read image alt text. For line 95, I would recommend we revert back to "LIFO" and then for lines like 97 and 111 (inside the brackets) we could add the spaces?

Reply:
I got confused. You were talking about alt text when in my earlier question, I was talking about the text inside the image itself. Haha. But sure. We could implement this.

Reply:
whoops! Sorry about that. Thanks for interjecting @Dusch4593 to explain what was going on behind the scenes 😆 Yeah, originally we were thinking about accessibility for users, but the speech function works fine without the spacing. I'll add in the comments for what Dusch recommended for the in-line images